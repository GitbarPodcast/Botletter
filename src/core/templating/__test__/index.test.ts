import renderer from '..';
import index from '../../../assets/htmlTemplate';

describe('templating', () => {
  it('it should render the template', async () => {
    const html = renderer(
      {
        articles: [],
        episode: { title: 'titolo', description: 'desc', image: 'image url', link: 'link_url' },
      },
      index as React.FC,
    );
    expect(html).toMatchSnapshot();
  });
});
