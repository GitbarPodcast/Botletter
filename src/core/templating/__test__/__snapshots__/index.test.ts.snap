// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`templating it should render the template 1`] = `"<html><body><h1 style=\\"font-size: 1.5em; text-align: center; color: palevioletred;\\">titolo <div>miao</div></h1></body></html>"`;
